### PR TITLE
Adding the official image for the contribute devfile / factory flow

### PR DESCRIPTION
**Pull request by ibuziuk**
### What does this PR do?
Adding the official image for the contribute devfile / factory flow
The file is meant to be used for the contribute devfile / factory link:

e.g. [Contribute](https://workspaces.openshift.com/f?url=https://github.com/redhat-developer/codeready-workspaces)

This repo is the best place for hosting it - we can not use the eclipse website for it  - https://github.com/eclipse/che-website/pull/54#discussion_r582867693

### What issues does this PR fix or reference?
N/A

#### Release Notes
N/A

#### Docs PR (if applicable)
N/A

### CODE REVIEW

File: assembly/branding/CRW-contribute.svg [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+       inkscape:connector-curvature="0"
+       d="m 1021.1,676.9 c -23.63,-2.35 -45.08,-11.06 -62.85,-24.26 -34.07,-25.31 -54.54,-67.14 -50.67,-112.36 0.12,-3.64 0.59,-7.2 1.36,-10.64 5.76,-25.76 28.76,-45.01 56.26,-45.01 7.6,0 14.85,1.47 21.49,4.14 9.2,2.88 18.45,5.56 27.75,8.05 46.48,12.46 94.09,20.23 142.1,24.35 0,0 0,0 0,-0.01 -65.67,-6.01 -127.88,-32.24 -178.04,-75.11 l 0.01,0.02 c -2.19,-1.87 -4.43,-3.67 -6.74,-5.38 -21.06,-15.66 -47.16,-24.93 -75.42,-24.93 -69.84,0 -126.46,56.59 -126.52,126.41 0,0.05 0,0.1 0,0.15 0,0.03 0,0.05 0,0.08 v 0 c 0,107.91 87.48,195.39 195.39,195.39 v 0 c 99.22,0 181.16,-73.95 193.72,-169.74 -9.11,67.18 -69.85,115.6 -137.84,108.85 z"
+       id="path8" />
+    <path
+       style="fill:#cb2026"
+       inkscape:connector-curvature="0"
+       d="m 965.2,347.01 v 0 c -20.76,0 -40.76,3.24 -59.53,9.24 -78.73,25.16 -135.76,98.87 -135.86,185.93 0.06,-69.83 56.68,-126.41 126.52,-126.41 28.26,0 54.36,9.27 75.42,24.93 l 6.73,5.36 c 50.16,42.87 112.38,69.1 178.04,75.11 0.77,0.07 1.54,0.14 2.31,0.2 0.15,0.01 0.3,0.03 0.45,0.04 0.06,0.01 0.13,0.01 0.19,0.01 -10.46,-98.04 -93.45,-174.41 -194.27,-174.41 z"
+       id="path10" />
+  </g>
```
Comment (nickboldt) [inline assembly/branding/CRW-contribute.svg:117]: rogue `</g>` tag?

Comment (ibuziuk) [reply assembly/branding/CRW-contribute.svg:117]: good catch I copied this one from https://github.com/redhat-developer-demos/quarkus-reactjs-postit-app/blob/master/factory-contribute.svg?short_path=e686622

Comment (ibuziuk) [reply assembly/branding/CRW-contribute.svg]: it is needed actually. The opening one is - https://github.com/redhat-developer/codeready-workspaces/pull/441/files#diff-757619813f684a8c994ab7773016d4ca3c79926ca2b47c60ee539ec0fdcfc717R89

Comment (nickboldt) [reply assembly/branding/CRW-contribute.svg]: GH UI shows it as a mistake, but you're right.

**Comment by nickboldt** [comment]
merged, cherrypicked to 2.7, and copyright date fixed

**Pull request merged**